packages/pxweb2-ui/src/lib/components/TableCard/TableCard.module.scss

```
@@ -0,0 +1,143 @@
+@use '../../../../style-dictionary/dist/scss/fixed-variables.scss' as fixvar;
+@use '../../../../style-dictionary/dist/css/variables.css' as var;
+@use '../../text-styles.scss';
+
+.tableCard {
+  display: flex;
+  border: 1px solid var(--px-color-border-subtle);
+  border-radius: var(--px-border-radius-large);
+  background: var(--px-color-surface-default);
+  color: var(--px-color-text-default);
+  padding: 20px;
+  position: relative;
+  cursor: pointer;
+
+  &:hover {
+    outline: 1px solid var(--px-color-border-subtle);
+
+    .title {
+      @extend .heading-underline;
+    }
+  }
+
+  &:focus-visible,
+  &:has(:focus-visible) {
+    border-radius: var(--px-border-radius-large);
+    outline: 3px solid var(--px-color-border-focus-outline);
+    outline-offset: 6px;
+    box-shadow: 0 0 0 3px var(--px-color-border-focus-boxshadow);
+  }
+}
+
+.iconWrapper {
+  min-width: 72px;
+  height: 72px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--border-radius-medium, 8px);
```

```suggestion
  border-radius: var(--px-border-radius-medium);
```
Probably just a simple oversight since this is the only CSS variable without the prefix and with a fallback, but this variable is undefined in Storybook and relies on the fallback.